tests/foreman/ui/test_docker.py

```diff
@@ -1151,95 +1154,95 @@ class DockerContainersTestCase(UITestCase):
+            organization=cls.organization).create()
+        cls.repo = entities.Repository(
+            content_type=u'docker',
+            docker_upstream_name=u'busybox',
+            product=entities.Product(organization=cls.organization).create(),
+            url=DOCKER_REGISTRY_HUB,
+        ).create()
+        cls.repo.sync()
+        content_view = entities.ContentView(
+            composite=False,
+            organization=cls.organization,
+        ).create()
+        content_view.repository = [cls.repo]
+        cls.content_view = content_view.update(['repository'])
+        cls.content_view.publish()
+        cls.cvv = content_view.read().version[0].read()
+        promote(cls.cvv, cls.lce.id)
+        cls.cr_internal = entities.DockerComputeResource(
+            name=gen_string('alpha'),
+            organization=[cls.organization],
+            url=settings.docker.get_unix_socket_url(),
+        ).create()
+        cls.cr_external = entities.DockerComputeResource(
+            name=gen_string('alpha'),
+            organization=[cls.organization],
+            url=settings.docker.external_url,
+        ).create()
+        cls.parameter_list = [
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Lifecycle Environment', 'value': cls.lce.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Content View', 'value': cls.content_view.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Repository', 'value': cls.repo.name},
+            {'main_tab_name': 'Image', 'sub_tab_name': 'Content View',
+             'name': 'Tag', 'value': 'latest'},
+        ]
 
-    @stubbed()
     @run_only_on('sat')
-    def test_create_container_local_compute_resource_read_log(self):
-        """@Test: Create a container in a local compute resource and read its
-        log
+    def test_create_container_compute_resource(self):
+        """@Test: Create containers for local and external compute resources
 
         @Feature: Docker
 
-        @Assert: The docker container is created in the local compute resource
-        and its log can be read
-
-        @Status: Manual
+        @Assert: The docker container is created for each compute resource
 
         """
+        with Session(self.browser) as session:
+            for compute_resource in (self.cr_internal, self.cr_external):
+                with self.subTest(compute_resource):
+                    make_container(
+                        session,
+                        org=self.organization.name,
+                        resource_name=compute_resource.name + ' (Docker)',
+                        name=gen_string('alphanumeric'),
+                        parameter_list=self.parameter_list,
+                    )
 
+    # Remove stubbed once https://github.com/theforeman/foreman-docker/pull/125
+    # is deployed
     @stubbed()
```

Comment:
That is long story, but thanks to Oggy, I finally can use BZ id rather than `stubbed()` deocrator